src/MLOps.NET.Azure/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("MLOps.NET.Azure.Tests")]
```

Comment:
Let's move these to the project file instead

Reply:
Fixed - didn't even know this was a feature with the new csproj format!